terraform/modules/fargate-service/alb.tf

```diff
@@ -0,0 +1,38 @@
+resource "aws_lb" "lb" {
+  name                             = "${var.name_prefix}-lb"
+  internal                         = true
```

Comment:
We need to be able to access this service via the internet, to the LB should not be internal